extensions/src/org/pentaho/platform/web/http/filters/PentahoWebContextFilter.java

```diff
@@ -369,6 +368,20 @@ private void printResourcesForContext( String contextName, OutputStream out, Htt
 
   }
 
+  public void printRequireJsCfgStart( OutputStream out ) throws IOException {
+
+    int waitTime;
+
+    try {
```

Comment:
org.pentaho.platform.engine.core.system.PentahoSystem uses org.pentaho.platform.engine.core.system.SystemSettings which already uses  cache for system variables. See  [https://github.com/pentaho/pentaho-platform/blob/0e9201a186de07d2e8693a4360613e14597a5b79/core/src/org/pentaho/platform/engine/core/system/SystemSettings.java#L80-L91] .  Then user may refresh variables clicking Tools->Refresh->System Settings  in user console. The refresh command [https://github.com/pentaho/pentaho-platform/blob/0e9201a186de07d2e8693a4360613e14597a5b79/core/src/org/pentaho/platform/engine/core/system/SystemSettings.java#L185-L187] only cleans the cache map and doesnt work with CacheManager. Whether it is necessary in this case to use CacheManager.